src/plugins/jqplot.donutRenderer.js

```diff
@@ -450,7 +454,10 @@
                 labelelem.css({left: x, top: y});
             }
         }
-               
+        if (this.totalLabel) {
+            var totalLabel = $('<div class="jqplot-data-label" style="position:absolute">' + this._totalAmount + '</div>').insertAfter(plot.eventCanvas._elem);
```

Comment:
Totally understand your reasoning. I would say keep it then as you did. We can then decide to make the improvement of avoiding inline styles dealing in a specific issue created for this. Alright?